pkg/util/membercluster_client.go

```diff
@@ -84,28 +92,36 @@ func NewClusterClientSetForAgent(clusterName string, client client.Client, clien
 }
 
 // NewClusterDynamicClientSet returns a dynamic client for the given member cluster.
-func NewClusterDynamicClientSet(clusterName string, client client.Client) (*DynamicClusterClient, error) {
+func NewClusterDynamicClientSet(clusterName string, client client.Client, clientOption *ClientOption) (*DynamicClusterClient, error) {
```

Comment:
@Poor12 I have a question. 
Assume we have two clients that are built by the `NewClusterDynamicClientSet`, they don't affect each other, right?

A := NewClusterDynamicClientSet(xxx)
B := NewClusterDynamicClientSet(xxx)

If both A and B are used to access the sam cluster, the maximum request could be 2 * QPS, right?

Reply:
ping. I still want to know the answer.